src/routes/console/organization-[organization]/settings/downloadDPA.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+    import { Box, CardGrid, Heading } from '$lib/components';
+    import { Button } from '$lib/elements/forms';
+    import { sdk } from '$lib/stores/sdk';
+
+    async function downloadPdf() {
+        const today = new Date().toISOString();
+        const prefs = await sdk.forConsole.account.getPrefs();
+        const newPrefs = { ...prefs, DPA: today };
+        sdk.forConsole.account.updatePrefs(newPrefs);
+
+        const baseUrl = window.location.origin;
+        const response = await fetch(`${baseUrl}/Appwrite_DPA.pdf`);
+        const blob = await response.blob();
+        const url = URL.createObjectURL(blob);
+        const link = document.createElement('a');
+        link.href = url;
+        link.download = 'Appwrite_DPA.pdf';
+        link.click();
+    }
+</script>
+
+<CardGrid>
+    <div>
+        <Heading tag="h6" size="7">Download DPA document</Heading>
+    </div>
+    <p class="text">
+        After downloading, have the DPA signed by your organization's compliance authority (e.g.,
+        CEO or Compliance Manager) and submit it to privacy@appwrite.io.
```

Comment:
```suggestion
        After downloading, have the DPA signed by your organization's compliance authority, such as your CEO or Compliance Manager, and submit it to privacy@appwrite.io.
```

Reply:
I would go as far as to omit the examples for CEO and compliance manager. I think it's for the organization to decide who this person should/can be.

Can we also make sure the email's a link?